pkg/edgeview/src/basics.go

```diff
@@ -143,6 +143,56 @@ func initOpts() {
 	}
 }
 
+// this script is automatically installed into your /tmp/download/bin
+// and can be used to run your kubectl with the download kubeconfig file
+// decrypted by the ssh private key
+const kubeConfdecrpytScript = `#!/bin/bash
```

Comment:
```suggestion
const kubeConfDecryptScript = `#!/bin/bash
```

Are we sure we want to be tied to `bash`?